File: Java/program93.java
// Input : 4
// Output :
/*
    1 2 3 4
    1 2 3 4
    1 2 3 4
    1 2 3 4
*/

import java.util.*;

class Pattern
{
    public void DisplayPattern(int iRow, int iCol)
    {
        int j = 0, i =0;
        for(i = 1; i <= iRow; i++)
        {
            for(j = 1 ; j<= iCol; j++)
            {
                System.out.print(j + "\t");
            }
            System.out.println();
        }
        


    }
}
class program93
{
    public static void main(String A[])
    {
        int iValue1 = 0, iValue2 = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter rows :");
        iValue1 = sobj.nextInt();

        System.out.println("Enter columnc :");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.DisplayPattern(iValue1,iValue2);
    }
}

File: Python/program93.py
# Input : 4
# Output :
'''
   * * * *
   # # # #
   * * * *
   # # # # 
'''

def Display(iRow,iCol):
    for i in range(1,iRow+1,1):
        for j in range(1,iCol+1,1):
            if((i % 2 ) == 0):
                print("#", end=" ")
            else:
                print("*", end=" ")
        print()
       
def main():

    print("Enter rows : ")
    iValue1 = int (input())
    
    print("Enter column : ")
    iValue2 = int (input())
    
    Display(iValue1,iValue2)
    
    
if __name__ == "__main__":
    main()